lib/scala/docs-generator/src/main/scala/org/enso/DocsGeneratorMain.scala

```diff
@@ -0,0 +1,124 @@
+package org.enso
+
+import java.io._
+import scala.util.Using
+import scala.io.Source
+import scalatags.Text.{all => HTML}
+import HTML._
+import docsgenerator.DocsGenerator._
+import docsgenerator.TreeOfCommonPrefixes._
+
+/** The Docs Generator script.
+  * Used to create HTML documentation for Enso Standard Library, and also JS
+  * files with react components for the "component reference" website.
+  */
+object DocsGeneratorMain extends App {
+
+  /// Files
+  val path = "./distribution/std-lib/Standard/src"
```

Comment:
Nope. You're doing the same thing we discussed with the python version and hardcoding a bunch of constants. Things like this should be accepted via the command-line (using our CLI args framework), not hard-coded in the source.